spec/features/sdr_deposit_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+RSpec.describe 'SDR deposit', type: :feature do
+  let(:start_url) { 'https://argo-stage.stanford.edu/' }
+  let(:api_url) { 'https://sdr-api-stage.stanford.edu' }
+  let(:source_id) { "testing:#{SecureRandom.uuid}" }
+  let(:apo) { 'druid:sc012gz0974' }
+  let(:collection) { 'druid:hv320hk4091' }
+  let(:catkey) { '10065784' }
+
+  before do
+    authenticate!(start_url: start_url, expected_text: 'Welcome to Argo!')
+  end
+
+  # This allows a login using credentials from the config gem.
```

Comment:
Is `the config gem` copypasta? I didn't think we used that gem in this codebase.